packages/docusaurus-plugin-search/theme/SearchModal/style.module.css

```diff
@@ -0,0 +1,63 @@
+.modal {
+  background: #f5f6f7;
```

Comment:
Take a look at Infima color variables after you've settled on a design. Will help make it easier for people to match this with the rest of their website.